kive/container/templates/container/batch_form.html

```diff
@@ -68,6 +68,16 @@ <h3>Container Runs:</h3>
             <div class="asf-main-search">
                 <div class="asf-active-filters"></div>
                 <input type="text" name="smart" class="asf-search-field">
+                <div class="la-ball-spin la-sm">
+                    <div></div>
+                    <div></div>
+                    <div></div>
+                    <div></div>
+                    <div></div>
+                    <div></div>
+                    <div></div>
+                    <div></div>
+                </div>
```

Comment:
The effect is beautiful, but I'm a little concerned about how much boilerplate is needed in each page, twice. Could the `PermissionsTable` class add all this, possibly into a div with an expected class?

Reply:
Agreed, it's a bit much. It's a little faster this way (dynamically inserting elements is expensive), but probably wouldn't result in a noticeable slowdown. But is there something we could do with Django templates instead? The markup for these all look pretty similar.

Reply:
It looks like the Django [include] tag would be useful here.

[include]: https://docs.djangoproject.com/en/3.0/ref/templates/builtins/#include

Reply:
Of course, the whole table is being built by dynamically inserting elements every time you refresh. Is dynamically inserting a few divs at page load even 1% of that? I suggest you look at the include tag, and take the performance win if it's really easy. If it's at all annoying, just go with the Javascript option.

Reply:
I moved the basic search field and the advanced submit button into templates, and it works well. I'll merge this branch.